site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<blockquote>two transfer learning strategies—<em>feature extraction</em> and <em>fine-tuning</em></blockquote><p> </p><p>I'm not sure these are separate enough to talk about as different *strategies*. You use an existing model to extract the features. You build a classifier on top of that. You need to train the classifier. You can fine-tune the freature-extractor.</p><p> </p><blockquote>previously trained on a&nbsp;typically&nbsp;large dataset for a particular task, such as&nbsp;image&nbsp;classification</blockquote><p> </p><p>This doesn't parse correctly.</p><p> </p><blockquote>A pre-trained model is a <a href="https://github.com/tensorflow/docs/raw/19d2699318803eab2c0c88f92cf072da887194fc/site/en/guide/saved_model.ipynb" target="_blank">saved model</a></blockquote><p> </p><p>A pre-trained model can be a saved model, but it could be other things too.</p><p> </p><blockquote>You will classify images of cats and dogs by using pre-trained models from <a href="https://tfhub.dev/" target="_blank">TensorFlow Hub</a></blockquote><p> </p><p>Would we gain anything to parameterize over the datasets (flowers or Cats and Dogs) or the model source (hub or keras-applications)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<blockquote><br></blockquote><blockquote>"two transfer learning strategies—<em>feature extraction</em>&nbsp;and&nbsp;<em>fine-tuning"</em></blockquote><blockquote>I'm not sure these are separate enough to talk about as different *strategies*.</blockquote><p>"two transfer learning approaches". LMKWYT</p>